src/test/resources/processor/IndexMappings.json

```diff
@@ -67,6 +67,13 @@
             }
           }
         }
+      },
+      "passage_embedding": {
+        "type": "knn_vector",
+        "dimension": 768
```

Comment:
this need to have a `lucene` as engine, otherwise it uses default which is `nmslib` and that fails, as nmslib is not part of the knn zip fopr integ tests @vibrantvarun

Reply:
Where is this field used by the way?